minutes/2021-08-10.md

```diff
@@ -0,0 +1,32 @@
+# Parsec Community Meeting (10/08/2021) 📋
+
+Minutes of previous meeting are stored
+[here](https://github.com/parallaxsecond/community/tree/main/minutes).
+
+## Attendance
+
+- Matt Davis
+- Paul Howard
+- Ionut Mihalcea
+- Hugues de Valon
+
+## Agenda
+
+- Update on the 0.8.0 release
+- Capability discovery and PSA Crypto
+
+## Minutes
+
+- Waiting on two maintainers for feedback.
+- Fine to not include Trusted Service provider now if the issues are only on the distributions side.
+- TS code not included in the source bundle generated by GH because it is a git submodule.
+- Possibility to create a branch of the `0.8.0` tag for future bugs.
+- Code freeze period as a courtesy window, we could release anyway at the end of it.
+- Code freeze remain in effect until beginning of September when the release will happen.
+- Other testers: Hugues, Pete, Wiktor, Paul (not worse since 0.7.2).
```

Comment:
Could you add details here about who's testing on what? Otherwise it sounds like "Paul is not worse since 0.7.2"